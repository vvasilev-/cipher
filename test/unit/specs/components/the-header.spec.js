/**
 * The external dependencies.
 */
import { shallow } from 'vue-test-utils';
import { createRenderer } from 'vue-server-renderer';

/**
 * The internal dependencies.
 */
import TheHeader from 'components/the-header';

describe('TheHeader', () => {
	it('should render properly', async () => {
		const wrapper = shallow(TheHeader);
		const renderer = createRenderer();
		const result = await renderer.renderToString(wrapper.vm);

		expect(result).toMatchSnapshot();
	});
});
