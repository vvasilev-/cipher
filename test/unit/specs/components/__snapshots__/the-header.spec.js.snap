// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheHeader should render properly 1`] = `"<header data-server-rendered=\\"true\\" class=\\"pv4\\"><h1 class=\\"f2 tc ttu tracked ma0\\">Check your password</h1></header>"`;
